conda/meta.yaml

```diff
@@ -25,61 +25,66 @@ source:
 
 requirements:
   build:
+    - {{ compiler('cxx') }}
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
     - cmake
-    - cppcheck [linux64]
     - conan>=1.45.0
-    - markupsafe>=1.1.1,<2.1.0  # see https://github.com/pallets/markupsafe/issues/284
-    - gxx_linux-64 11.1.* [linux64]
+    - cppcheck [linux64]
     - git
+    - markupsafe>=1.1.1,<2.1.0  # see https://github.com/pallets/markupsafe/issues/284
     - ninja
     - python
+  host:
+    - python
     - tbb-devel=2020.2.*
   run:
     - confuse
-    - numpy>=1.20.0
-    - python
+    - numpy
     - pyyaml
     - tbb=2020.2.*
     - graphlib-backport # for python < 3.9
 
+# Only the cmake-package-test works when cross compiling.
+# But it is not very important at the moment and
+# leaving out the entire test section speeds up conda build
+{% if build_platform == target_platform %}
 test:
   imports:
     - scipp
   requires:
+    - {{ compiler('cxx') }}
     - cmake
     - conan>=1.45.0
-    - markupsafe>=1.1.1,<2.1.0  # see https://github.com/pallets/markupsafe/issues/284
-    - gxx_linux-64 11.1.* [linux64]
     - h5py
     - hypothesis
     - ipympl
     - ipywidgets
     - markupsafe>=1.1.1,<2.1.0  # see https://github.com/pallets/markupsafe/issues/284
     - matplotlib-base
+    - pandas
     - pytest
     - pythreejs
     - python-graphviz
-    - xarray
-    - pandas
     - scipy>=1.7.0
     - tbb-devel=2020.2.*
+    - xarray
   files:
     - cmake-package-test/
   source_files:
     - tests/
   commands:
     - python -m pytest -v tests
     - python cmake-package-test/build.py
+{% endif %}
 
 build:
   # Build number is number of Git commits since last tag, if that fails use 0
   number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
   script:
-    - cmake --preset package-linux [linux]
-    - cmake --preset package-macos [osx]
-    - cmake --preset package-windows [win]
+    # Passing Python_EXECUTABLE ensures that cmake sees the host interpreter instead of build.
```

Comment:
This one is a little weird. Can you add a link to where you found you needed to do this to get it to work?

Reply:
I originally added it in [d2e06f8](https://github.com/scipp/scipp/pull/2518/commits/d2e06f88a5c7979012c4158ae44468cd8f6b0ff3) because the previous build failed: 
https://github.com/scipp/scipp/runs/5751095950?check_suite_focus=true#step:5:2328
with
```
Traceback (most recent call last):
  File "/usr/share/miniconda/envs/test/conda-bld/scipp_1648630531422/test_tmp/run_test.py", line 2, in <module>
    import scipp
  File "/usr/share/miniconda/envs/test/conda-bld/scipp_1648630531422/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.8/site-packages/scipp/__init__.py", line 24, in <module>
    from .core import __version__
  File "/usr/share/miniconda/envs/test/conda-bld/scipp_1648630531422/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.8/site-packages/scipp/core/__init__.py", line 6, in <module>
    from .._scipp import _debug_
ModuleNotFoundError: No module named 'scipp._scipp'
Tests failed for scipp-0.13.0-py38h2bc3f7f_6.tar.bz2 - moving package to /usr/share/miniconda/envs/test/conda-bld/broken
WARNING:conda_build.build:Tests failed for scipp-0.13.0-py38h2bc3f7f_6.tar.bz2 - moving package to /usr/share/miniconda/envs/test/conda-bld/broken
export PREFIX=/usr/share/miniconda/envs/test/conda-bld/scipp_1648630531422/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
TESTS FAILED: scipp-0.13.0-py38h2bc3f7f_6.tar.bz2
export SRC_DIR=/usr/share/miniconda/envs/test/conda-bld/scipp_1648630531422/test_tmp
import: 'scipp'
Error: Process completed with exit code 1.
```

According to this: https://github.com/scipp/scipp/runs/5751095950?check_suite_focus=true#step:5:335
```
-- Found Python: $BUILD_PREFIX/bin/python3.9 (found suitable version "3.9.12", minimum required is "3.8") found components: Interpreter 
```
It found Python 3.9 which was used in the build env. But the host env has 3.8 (specified using `--python=3.8` in `conda build`).